tests/net/ieee802154/fragment/src/main.c

```diff
@@ -244,8 +245,10 @@ static struct net_pkt *create_pkt(struct net_fragment_data *data)
 	remaining = data->len;
 
 	len = NET_UDPH_LEN + remaining;
-	/* length is not set in net_fragment_data data pointer, calculate and set
-	 * in ipv6, udp and in data pointer too (it's required in comparison) */
+/*
```

Comment:
Original indentation is correct

Reply:
Threre was error from checkpatch thats why I changed this. But I will make something similar now